src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java

```diff
@@ -82,13 +84,31 @@ public CommandResult execute(Model model) throws CommandException {
         Appointment editedAppointment = createEditedAppointment(appointmentToEdit, editAppointmentDescriptor,
```

Comment:
<img width="948" alt="image" src="https://github.com/AY2324S1-CS2103T-W09-3/tp/assets/64828959/89945b04-4f75-453b-9075-9b8deeeac8b4">


Getting seedu.address.model.timeslots.exceptions.TimeSlotNotFoundException when editing appointment time

Reply:
similarly will require the steps to recreate the issue